src/Seksjon/Utdanning.elm

```diff
@@ -0,0 +1,193 @@
+module Seksjon.Utdanning exposing (Model, ModelInfo, Msg, Samtale(..), SamtaleStatus(..), init, meldingsLogg, update, viewUtdanning)
```

Comment:
Her exposes det litt for mye. Greia med moduler er jo at man bare skal expose det som er nødvendig, og ikke noe mer, sånn at de andre modulene har så få avhengigheter til deg som mulig. Her ville jeg ikke exposet `ModelInfo` og ikke `Samtale(..)`.